pkg/operator/operator.go

```diff
@@ -77,7 +77,6 @@ func New(config operatorconfig.Config, kubeConfig *rest.Config) (*Operator, erro
 			operatorcontroller.DefaultOperandNamespace,
 			operatorcontroller.DefaultCanaryNamespace,
 			operatorcontroller.GlobalMachineSpecifiedConfigNamespace,
-			"",
 			operatorcontroller.SourceConfigMapNamespace,
 			operatorcontroller.GlobalUserSpecifiedConfigNamespace,
```

Comment:
Do we need `operatorcontroller.GlobalUserSpecifiedConfigNamespace` twice?

Reply:
Oh good catch! No, there is no reason to have them twice.

Reply:
Done.